inspirehep/config.py

```diff
@@ -1496,6 +1504,45 @@ def _(x):
             "default_order": 'desc',
             "order": 2,
         },
+        "core": {
+            "title": 'Core (Desc)',
+            "fields": [
+                {
+                    '_extra_data.relevance_prediction.scores.CORE':
+                        {
```

Comment:
Don't add cognitive load when you don't need to add it. 3 lines above you have a list, and then you indent the objects inside. Here you indent in exactly the same way, but we're not inside of a list!

Small things like this add up. See: https://en.wikipedia.org/wiki/Stroop_effect